extensions/merge-conflict/src/mergeConflictParser.ts

```diff
@@ -58,7 +58,7 @@ export class MergeConflictParser {
 				currentConflict.commonAncestors.push(line);
 			}
 			// Are we within a conflict block and is this a splitter? =======
-			else if (currentConflict && !currentConflict.splitter && line.text.startsWith(splitterMarker)) {
+			else if (currentConflict && !currentConflict.splitter && line.text.match(splitterMarkerRegex)) {
```

Comment:
Why are you using a regex rather than a simple equality test?

Reply:
You are right. Comment in #95040 led me to think that a regex was required to ensure correct line ending detection.